gapic/src/main/com/google/gapid/perfetto/views/StyleConstants.java

```diff
@@ -196,116 +184,74 @@ public static Colors dark() {
             DARK_TIMELINE_RULER,
             DARK_VSYNC_BACKGROUND,
             DARK_TEXT_MAIN,
-            DARK_TEXT_ALT,
-            DARK_TEXT_INVERTED_MAIN,
-            DARK_TEXT_INVERTED_ALT);
+            DARK_TEXT_ALT);
     }
   }
 
-  private static final HSL GRAY_COLOR = new HSL(0, 0, 62);
-
-  /*
-   * Definition and API for colors in GAPID.
-   * */
-  public static class Palette {
-    public static enum BaseColor {
-      GREY(new HSL(202, 25, 67)),
```

Comment:
Would still prefer to have these names somewhere, even in comment, for easy correlation in the future.

Reply:
Done.